docs/en/02_Developer_Guides/14_Files/07_File_Usage.md

```diff
@@ -0,0 +1,63 @@
+---
+title: File usage
+summary: See file usage and customising the file "Used on" table
+icon: compress-arrows-alt
+---
+
+# File Usage
+
```

Comment:
Maybe add a bit of an intro about what the feature does.

```suggestion

CMS users can view where a file is used by accessing the "Used On" tab in the file editor. This feature allows them to identify what DataObjects depend on the File. 

```

Reply:
Added